main.go

```diff
@@ -3,42 +3,44 @@ package main
 import (
 	"log"
 	"net/http"
+	"os"
 
-	"github.com/gorilla/mux"
-	"github.com/joyent/triton-service-groups/groups"
-	"github.com/joyent/triton-service-groups/templates"
+	"github.com/gorilla/handlers"
```

Comment:
I wouldn't swap this out now, but I have developed an affinity for `Gin` over `gorilla`.  Gorilla's a workhorse, but a bit dated in its structure and design.  This is just something to keep in mind for the future.  I'll share some code using Gin that will, I think, make you happy.

Reply:
I spoke about this with @cheapRoc earlier - we will more than likely roll our own logging handler soon